### PR TITLE
Throw UOE when calling delete on an adder

### PR DESCRIPTION
**Pull request by SplotyCode**
Signed-off-by: david <davidscandurra@gmail.com>

### CODE REVIEW

File: redisson/src/main/java/org/redisson/RedissonBaseAdder.java [hunk 1]
```diff
@@ -269,6 +269,21 @@ protected String getCounterName(String id) {
         return result;
     }
 
+    @Override
+    public boolean delete() {
+        throw new UnsupportedOperationException();
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/RedissonBaseAdder.java:274]: Please remove this. Since it always call `deleteAsync()` method

**Review by mrniko — CHANGES_REQUESTED** [review]

**Review by mrniko — APPROVED** [review]

**Comment by mrniko** [comment]
any update?

**Comment by SplotyCode** [comment]
History is a bit messed up. As I tried to sign past commits. I will fix it tomorrow

**Comment by mrniko** [comment]
any update?

**Pull request closed without merging**